src/components/modals/InsufficientFeesModal.tsx

```diff
@@ -12,26 +12,38 @@ import { getCurrencyCode } from '../../util/CurrencyInfoHelpers'
 import { getUkCompliantString } from '../../util/ukComplianceUtils'
 import { roundedFee } from '../../util/utils'
 import { ButtonsView } from '../buttons/ButtonsView'
+import { Airship } from '../services/AirshipInstance'
 import { Paragraph } from '../themed/EdgeText'
 import { EdgeModal } from './EdgeModal'
 
 interface Props {
-  bridge: AirshipBridge<void>
   coreError: InsufficientFundsError
+  countryCode?: string
   navigation: NavigationBase
   wallet: EdgeCurrencyWallet
 
-  countryCode?: string
   // Called when the user wants to swap.
   // The default behavior is to navigate to the swap scene,
   // but the swap scene itself needs a different behavior here.
   onSwap?: () => void
 }
 
+interface InsufficientFeesModalProps extends Props {
+  bridge: AirshipBridge<void>
+}
+
 /**
  * Show this modal when the wallet doesn't have enough funds to cover fees.
+ * If UK and swaps are disabled, this is a noop
  */
-export function InsufficientFeesModal(props: Props) {
+export const showInsufficientFeesModal = async (props: Props) => {
+  const { countryCode } = props
+
+  if (config.disableSwaps === true && countryCode === 'GB') return
```

Comment:
In this case, would it make sense to show a simple `ButtonsModal` with a message like "Insufficient funds to cover fees". Otherwise, UK users would try to send tokens and just silently fail with no feedback. We can't help the user solve the problem, since we can't offer any swaps or buys, but we can at least tell them that there is a problem.

Reply:
I reasoned that the scene itself would show the error, but you're right that keeping some sort of modal keeps the same flow and give more room for modified messaging